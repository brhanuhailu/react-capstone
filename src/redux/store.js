import { configureStore } from '@reduxjs/toolkit';
import cryptoSlice from './crypto/cryptoSlice';

const store = configureStore({
  reducer: {
    cryptoData: cryptoSlice,
  },
});

export default store;
