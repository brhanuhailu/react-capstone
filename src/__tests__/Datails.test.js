import renderer from 'react-test-renderer';
import { BrowserRouter } from 'react-router-dom';
import { Provider } from 'react-redux';
import Details from '../components/detail';
import store from '../redux/store';

describe('Detail', () => {
    it('renders correctly', () => {
        const tree = renderer
        .create(
            <Provider store={store}>
            <BrowserRouter>
                <Details />
            </BrowserRouter>
            </Provider>,
        )
        .toJSON();
        expect(tree).toMatchSnapshot();
    });
    });