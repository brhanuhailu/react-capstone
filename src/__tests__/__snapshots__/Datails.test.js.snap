// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Detail renders correctly 1`] = `
<div
  className="details-container"
/>
`;
